galasa-ui/messages/de.json

```diff
@@ -4,7 +4,7 @@
   },
   "Breadcrumb": {
     "home": "Startseite",
-    "users": "Benutzer",
+    "users": "Benutzeren",
```

Comment:
I know this was raised by @techcobweb, but I'm certain "Benutzeren" isn't a word. "Benutzer" can be singular and plural - e.g. "der Benutzer" (singular), "die Benutzer" (plural).

We can keep it as it was previously - same applies throughout this PR.